engine/api/workflow_run.go

```diff
@@ -485,7 +485,8 @@ func (api *API) getWorkflowCommitsHandler() Handler {
 
 		commits, _, errC := workflow.GetNodeRunBuildCommits(api.mustDB(), api.Cache, proj, wf, nodeName, wfRun.Number, wfNodeRun, nodeCtx.Application, nodeCtx.Environment)
 		if errC != nil {
-			return sdk.WrapError(errC, "getWorkflowCommitsHandler> Unable to load commits")
+			// as workflow.GetNodeRunBuildCommits can return a 401 error, we avoid here to return 401 on UI too
+			return sdk.WrapError(sdk.ErrNoReposManagerClientAuth, "getWorkflowCommitsHandler> Unable to load commits: %v", errC)
```

Comment:
Is a Forbidden status code okay ? (asking for a friend)

Reply:
yes: 	ErrNoReposManagerClientAuth               = Error{ID: 44, Status: http.StatusForbidden}